lib/runtime-c-api/src/lib.rs

```diff
@@ -180,7 +181,7 @@ pub unsafe extern "C" fn wasmer_validate(
 /// and `wasmer_last_error_message` to get an error message.
 #[no_mangle]
 pub unsafe extern "C" fn wasmer_memory_new(
-    mut memory: *mut *mut wasmer_memory_t,
+    memory: *mut *mut wasmer_memory_t,
```

Comment:
`memory` contains a mutable pointer, but the binding itself is not mutable. `rustc` generates a warning for that.